src/main/java/us/hebi/sass/PlatformUtil.java

```diff
@@ -52,7 +54,9 @@ public static void downloadAndExtractArchive(URL url, Path destinationDirectory)
         Path tmpArchive = destinationDirectory.resolve(createTemporaryArchiveName(url.toExternalForm()));
```

Comment:
I think the temporary archive `archive.zip` is the file that creates problems. I think your fix might have a similar issue as the original code.

Reply:
The idea behind using atomic move was to make this operation atomic on FS level. It should prevent running into this issue if it was the maven process who locked the file. 
I believe we have at least two possible solutions that should ultimately solve the problem:
1. Go for unique file name for every run, as you proposed
2. Remove the target file.

I've just did the later one but, I believe it can still cause other problems - the delete operation may fail. 
I will rework it.